src/components/home.jsx

```diff
@@ -6,6 +6,9 @@ import Filter from './filter';
 import { Container, Row } from 'bootstrap-4-react';
 import { Progress } from 'bootstrap-4-react';
 import ScrollToTop from './scrollToTop';
+import Board from './board.jpg';
```

Comment:
could you please move all the images in `/src/component` to `/src/assets`? And update the paths in the files
ps: you have to make a new directory `assets`

Reply:
done @Namanl2001 .